llvm/test/CodeGen/RISCV/GlobalISel/legalizer/legalize-bswap-rv64.mir

```diff
@@ -2,7 +2,9 @@
 # RUN: llc -mtriple=riscv64 -run-pass=legalizer %s -o - \
 # RUN:   | FileCheck %s --check-prefix=RV64I
 # RUN: llc -mtriple=riscv64 -mattr=+zbb -run-pass=legalizer %s -o - \
-# RUN:   | FileCheck %s --check-prefix=RV64ZBB
+# RUN:   | FileCheck %s --check-prefix=RV64ZBKB
```

Comment:
The RV32 check-prefix mentions both extensions, but this only mentions one.

Reply:
oops. Thanks, fixed.